### ISSUE TITLE
I can't run this code.

### ISSUE THREAD

**Issue by Aaron2Woo** [mohamadtaj/GreenSwarm#1]
I can't run this code. Is this code complete?
For example, in gs_peer.py, line 77 of the code self.model.set_weights(self.parameters), the set_weights method is not defined.

**Comment by arsamtaj** [comment]
Yes, the code is complete. The "set_weights" method is already available in Keras. If you have Keras installed, there shouldn't be any problems. If not, please try installing Tensorflow and Keras. I have added the list of all requirements to the README file. Please let me know if this doesn't fix the problem.
